inginious/frontend/static/js/studio.js

```diff
@@ -17,8 +17,7 @@ function studio_load(data)
     });
 
     // Hacky fix for codemirror in collapsable elements
-    var collapsable = $('#tab_subproblems').find('.collapse');
-    collapsable.on('show.bs.collapse', function()
+    $('.card').on('show.bs.collapse',function()
```

Comment:
Wouldn't be more appropriate to keep the ``$("#tab_subproblems")`` selector here, to avoid any redraw when unnecessary ?